pageserver/src/layered_repository/layer_map.rs

```diff
@@ -42,62 +41,24 @@ pub struct LayerMap {
     /// All in-memory layers, ordered by 'oldest_pending_lsn' of each layer.
```

Comment:
Should this comment mention `generation` too?

Reply:
added